djangocms_text_ckeditor/tests/frontend/integration/smoke.js

```diff
@@ -171,11 +173,12 @@ casper.test.begin('CKEditor loads correctly in collapsed fieldset', function (te
             this.click('input[type=submit]');
         })
         .waitForSelector('.success', function () {
-            test.assertSelectorHasText('.success', 'The pizza "Pizza object" was added successfully.', 'Pizza added');
+            test.assertSelectorHasText('.success', 'was added successfully.', 'Pizza added');
+            test.assertSelectorHasText('.success', 'Pizza object (2)', 'Pizza added');
```

Comment:
```suggestion
            test.assertSelectorHasText('.success', 'Pizza object', 'Pizza added');
```
that way it would be consistent across django versions (<2.0 didn't have the number in the name)